pkgs/tools/misc/rig/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl } :
+
+stdenv.mkDerivation rec {
+  version = "1.11";
+  name = "rig-${version}";
+
+  src = fetchurl {
+    url = "https://ayera.dl.sourceforge.net/project/rig/rig/${version}/rig-${version}.tar.gz";
+    sha256 = "1f3snysjqqlpk2kgvm5p2icrj4lsdymccmn3igkc2f60smqckgq0";
+  };
+  patches = [ ./rig_1.11-1.diff ];
+
+  meta = {
+    homepage = http://rig.sourceforge.net/; 
+    description = "Random identity generator";
+    longDescription = ''
+      RIG (Random Identity Generator) is a free replacement for a shareware
+      program out there called 'fake'. It generates random, yet real-looking,
+      personal data. It is useful if you need to feed a name to a Web site,
+      BBS, or real person, and are too lazy to think of one yourself. Also,
+      if the Web site/BBS/person you are giving the information to tries to
+      cross-check the city, state, zip, or area code, it will check out.
+      Note: diff modified from Debian: Norbert Veber <nveber@debian.org>
```

Comment:
This note belongs closer to `patches`. Could you also include the reference to the original patch?

Reply:
Where is best? Just use comments? Or is there a more appropriate structured place to put miscellaneous notes?